gtest_parallel_tests.py

```diff
@@ -446,5 +446,207 @@ def root():
                                    os.path.join(root(), 'c', 'd', 'bin'),
                                    'Test.case', 1))
 
+
+class TestFindTests(unittest.TestCase):
+  class MockOptions(object):
```

Comment:
Should these be under this test case? I'd expect mock options to be something generic that we can reuse?